src/Illuminate/Mail/MailManager.php

```diff
@@ -72,6 +72,17 @@ public function mailer($name = null)
         return $this->mailers[$name] = $this->get($name);
     }
 
+    /**
+     * @param string $name
```

Comment:
Missing description and incorrect spacing

Reply:
@GrahamCampbell how can I run style-linting locally?

Reply:
Those spaces are not enforced by the code style fixing service, which is why I left the review comment, and you need to write an actual description for the method. This cannot be automated.

Reply:
pushed a commit which addresses your comment